PurchasesCoreSwift/Support/ManageSubscriptionModalHelper.swift

```diff
@@ -0,0 +1,166 @@
+//
+//  Copyright RevenueCat Inc. All Rights Reserved.
+//
+//  Licensed under the MIT License (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      https://opensource.org/licenses/MIT
+//
+//  ManageSubscriptionModalHelper.swift
+//
+//  Created by Andrés Boedo on 16/8/21.
+
+import StoreKit
+
+enum ManageSubscriptionsModalError: Error {
+
+    case couldntGetPurchaserInfo(message: String)
+    case couldntGetWindowScene
+    case storeKitShowManageSubscriptionsFailed(error: Error)
+    case couldntGetAppleSubscriptionsURL
```

Comment:
this should in practice never happen, it could be replaced with an implicitly unwrapped optional in SystemInfo